### DISCUSSION TITLE
502 Bad Gateway - Nginx

### DISCUSSION THREAD

**Discussion by estebaninformatica** [weather-gov/api#612] [Q&A]
We often get 502 error when calling API, does anyone know how to fix ?

https://api.weather.gov/alerts/active
https://api.weather.gov/alerts/active?zone=MDC033

**Comment by scadergit** [comment]
500s unfortunately expected, but 502 would be news to me. Will you attach the full response (specifically the correlation id also found in the headers).

**Reply by estebaninformatica** [reply]
It's solved already ! But this nginx error lasted 30 min.

![image](https://user-images.githubusercontent.com/2387344/212928562-2ac11b27-6d18-45e7-8a68-f7e9ead2f8a4.png)

**Reply by scadergit** [reply]
Oh, you meant literally nginx. Didn't catch that in the title.

**Reply by akrherz** [reply]
I'm guessing this was to blame
```
SENIOR DUTY METEOROLOGIST NWS ALERT ADMINISTRATIVE MESSAGE
NWS NCEP CENTRAL OPERATIONS COLLEGE PARK MD
1331Z TUE JAN 17 2023


...COLLEGE PARK NETWORK OUTAGE/RESTORATION...

College Park Network problems started around 1120Z. NCO
investigated and Network
came backup at 1244Z.  The following applications were impacted
for 84min: API, FTPPRD, GISC, IRIS, MADIS, TGFTP, FTPS-IN, and
RADAR3.. IRIS was impacted for 2 hours due to a corrupt Database.
 IRIS was successfully failed over to Boulder and is back up.

Please visit https://www.nco.ncep.noaa.gov/status/messages/
for further updates.
```
